backend/control_center/notification/models.py

```
@@ -16,6 +16,12 @@ class Notification(models.Model):
     user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
     message = models.TextField()
     is_read = models.BooleanField(default=False)
+    URGENCY_CHOICES = [
+        ("low", "low"),
+        ("medium", "medium"),
+        ("high", "high"),
+    ]
```

Ruff (0.13.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)